src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -278,6 +281,27 @@ class WorkflowViewContent extends Component {
     ])
   }
 
+  downloadJson(key) {
+    const { modifiedConfig } = this.state
+    const blob = new Blob([JSON.stringify(modifiedConfig[key])], { type: 'application/json' })
+    FileSaver.saveAs(blob, `${key}.json`)
+  }
+
+  async uploadJson(key, files) {
+    try {
+      const text = await Utils.readFileAsText(files[0])
```

Comment:
Can this take `files[0]` as its argument, so it doesn't have to know to look into the array?

Reply:
Sure, though it just moves the array deref from one place to another.